Missing config.h

I'm trying to compile this again from source and I'm getting what's even missing here.


```
[ 27%] Built target obs-linuxbrowser
[ 36%] Building CXX object CMakeFiles/browser_shared.dir/src/browser/browser-app.cpp.o
In file included from /home/alkaris/obs-linuxbrowser/src/browser/browser-app.hpp:25:0,
                 from /home/alkaris/obs-linuxbrowser/src/browser/browser-app.cpp:29:
/home/alkaris/obs-linuxbrowser/src/browser/browser-client.hpp:24:10: fatal error: config.h: No such file or directory
 #include "config.h"
          ^~~~~~~~~~
compilation terminated.
CMakeFiles/browser_shared.dir/build.make:86: recipe for target 'CMakeFiles/browser_shared.dir/src/browser/browser-app.cpp.o' failed
make[2]: *** [CMakeFiles/browser_shared.dir/src/browser/browser-app.cpp.o] Error 1
CMakeFiles/Makefile2:104: recipe for target 'CMakeFiles/browser_shared.dir/all' failed
make[1]: *** [CMakeFiles/browser_shared.dir/all] Error 2
Makefile:129: recipe for target 'all' failed
make: *** [all] Error 2
```

I have everything I can think of installed, and I have whats listed in the dependencies installed too. I don't get why it's not able to find `config.h` when it should be there.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -54,7 +54,7 @@ set(CMAKE_CXX_STANDARD_REQUIRED true)
 # Ensure a consistent command chain on all systems
 set(CMAKE_CXX_EXTENSIONS false)
 
-include_directories(src ${PROJECT_BUILD_DIR}/src ${OBS_INCLUDE_DIR} ${CEF_INCLUDE_DIR})
+include_directories(src ${PROJECT_BINARY_DIR}/src ${OBS_INCLUDE_DIR} ${CEF_INCLUDE_DIR})
 
 set(PLUGIN_SOURCES
     src/plugin/main.c

